Cargo.toml

```diff
@@ -26,7 +26,6 @@ arrayref = "0.3"
 bincode = "1.0"
 bitflags = "1"
 byteorder = "1"
-hashbrown = { version = "0.6", optional = true }
```

Comment:
Also, this is unfortunate, but it the raw entry API doesn't exist for BTreeMap, which we need to use for sorted databases.